source/Octopus.Tentacle.Tests/Capabilities/CapabilitiesServiceV2Fixture.cs

```diff
@@ -18,8 +20,27 @@ public async Task CapabilitiesAreReturned()
             capabilities.Should().Contain("IScriptService");
             capabilities.Should().Contain("IFileTransferService");
             capabilities.Should().Contain("IScriptServiceV2");
-            capabilities.Should().Contain("IScriptServiceV3Alpha");
-            capabilities.Count.Should().Be(4);
+            capabilities.Count.Should().Be(3);
+
+            capabilities.Should().NotContainMatch("IKubernetesScriptService*");
+        }
+
+        [Test]
+        public async Task OnlyKubernetesScriptServicesAreReturnedWhenRunningAsKubernetesAgent()
+        {
+            Environment.SetEnvironmentVariable(KubernetesConfig.NamespaceVariableName, "ABC");
```

Comment:
We need to change the Kubernetes configuration to be testedable without environment variables, but will do that as a separate PR

Reply:
Ideally yes but many of the `Tentacle.Tests` (distinct from integration tests) suffer from environment variables being used e.g. within tentacle when a script is run :( 

Its frankly a bit of a mess.